### DISCUSSION TITLE
How to get HEVC/X265 60fps in anime?

### DISCUSSION THREAD

**Discussion by softsubs** [staxrip/staxrip#679] [Q&A]
hello how are you guys, I am new to staxrip and want to encode anime in 60fps but I don't know how to make it, can you teach me. Thank you in advance
sample image > https://imgur.com/a/fFoIJKx

**Comment by ghost** [comment]
do you want to create interpolated frames so your resulting video is 60fps and smooth ?
What is the fps of the source ?

**Reply by softsubs** [reply]
yes it's possible, because I want to produce and view videos with action scenes at 60fps, a source that I often take from subsplease.
can you help teach me to make it an example with episode 19 of kimetsu no yaiba which I took from horriblesubs.

**Reply by ghost** [reply]
But what is your original fps ?  23.976 ? 24 ? 29.97 ?

Anyway, in the filters, you can chose:
Filters > Frame Rate > SVPFlow   : this will double the frame rate and give you a very nice and smooth result
OR
Filters > Frame Rate > InterFrame : this will give you the fps you want. For 60fps, you must type, when asked, fpsnum = 60 and fpsden = 1.

**Reply by softsubs** [reply]
original fps 23.976 (24000/1001) FPS

In which filter should I use AVS Filters or VS Filters?

**Reply by ghost** [reply]
So SVPFlow will give you 47.9 fps.
And if you really want 60 fps use interframe, but it's slower.

You can choose AVS or VS.
Both are good. Maybe VS will be slightly faster.

**Reply by softsubs** [reply]
ok, I will try it in SVPFlow first using VS Filters,

And for interframe, is this step correct> VS Filters> Add> Frame Rate> InterFrame> NewNum 60> NewDen 1> Tuning Smooth> Algorithm Smoothest> GPU True or False? i am using gtx 1650 super which should i choose?

**Reply by ghost** [reply]
GPU: true

Here you'll find all interframe parameters explanation:
https://www.spirton.com/uploads/InterFrame/InterFrame2.html

In general, for filters google them like this : avisynth interframe and you'll find the explanation. For more information on this filter and other filter you have to search on google !

I recommend you use SVFLow: it is faster, and it gives very good results !

**Reply by softsubs** [reply]
Sample https://imgur.com/a/GBGMFJb

I tried it on SVPFlow.
When I select Replace and Insert video, it can't be entered as an example like in the picture. do I need to enter the video first and then start the settings and won't an error occur?

But if I choose Add it can.

**Reply by ghost** [reply]
Of course, in the screenshot you made, it appears you were attempting to replace the SOURCE filter with SVP Flow.
Careful where you click!
You can't replace the SOURCE filter because the SOURCE filter opens the video...
And this is why StaxRip tells you that you can't...

**Reply by chowderhead** [reply]
I encode 60fps video using NvencC and ChangeFPS(60) as the LAST command on left side window. with AVIsynth and usually watch for audio that goes out of sync.  If the audio goes out of sync just copy mux it to avoid this problem and you should get 60fps no issues.  Ive had issues with out of sync audio but solved by just copying audio with this single command at end of a AVIsynth script rather than VB.  It creates video at about 90fps on a 1650 since I have one as well as a rtx2070.
